CollisionOperator/CollisionProcessors/neutronCEimp_class.f90

```diff
@@ -162,13 +174,21 @@ subroutine init(self, dict)
       if (self % maxWgt < 2 * self % minWgt) call fatalError(Here,&
               'Upper weight bound must be at least twice the lower weight bound')
     end if
+
     if (self % implicitAbsorption) then
       if (.not.self % roulette) call fatalError(Here,&
          'Must use Russian roulette when using implicit absorption')
       if (.not.self % implicitSites) call fatalError(Here,&
          'Must generate fission sites implicitly when using implicit absorption')
     end if
 
+    ! Sets up the weight windows field
+    if (self % weightWindows) then
+      name = 'WeightWindows'
```

Comment:
Is this a global constant name? 
Perhaps we should have it as a global constant if multiple parts of the code will be trying to use it.

Reply:
`universal variables` seems like a best place (for now)